### PR TITLE
fix some typo under blogs/

### PR DESCRIPTION
**Pull request by digger-yu**
fix some typo under blogs/

detail info
	modified:   blogs/comm-opt/README.md
	modified:   blogs/deepspeed-fastgen/README.md
	modified:   blogs/deepspeed-offloadpp/README.md
	modified:   blogs/deepspeed-triton/README.md
	modified:   blogs/deepspeed-ulysses/README.md
	modified:   blogs/deepspeed-visualchat/10-03-2023/README-Japanese.md
	modified:   blogs/deepspeed-visualchat/10-03-2023/README.md

### CODE REVIEW

File: blogs/deepspeed-fastgen/README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ To address these shortcomings, we propose a novel prompt and generation composit
 
 ## 3. Dynamic SplitFuse: A Novel Prompt and Generation Composition Strategy<a name="technical-approach"></a>
 
-DeepSpeed-FastGen is built to leverage continuous batching and non-contiguous KV caches to enable increased occupancy and higher responsivity for serving LLMs in the data center, similar to existing frameworks such as TRT-LLM, TGI, and vLLM. In order to achieve a new level of performance, DeepSpeed-FastGen introduces SplitFuse which leverages dynamic prompt and generation decomposition and unification to further improve continuous batching and system throughput.
+DeepSpeed-FastGen is built to leverage continuous batching and non-contiguous KV caches to enable increased occupancy and higher responsively for serving LLMs in the data center, similar to existing frameworks such as TRT-LLM, TGI, and vLLM. In order to achieve a new level of performance, DeepSpeed-FastGen introduces SplitFuse which leverages dynamic prompt and generation decomposition and unification to further improve continuous batching and system throughput.
```
Comment (loadams) [inline blogs/deepspeed-fastgen/README.md:65]: @digger-yu - thanks for doing this.  I suspect that this was supposed to be "responsivity"

Comment (loadams) [inline blogs/deepspeed-fastgen/README.md:65]: @digger-yu - thanks for doing this.  I suspect that this one was supposed to be "responsivity" though

**Review by loadams — CHANGES_REQUESTED** [review]

File: blogs/deepspeed-fastgen/README.md (hunk 1 above)
Comment (digger-yu) [reply blogs/deepspeed-fastgen/README.md]: revert

Comment (loadams) [reply blogs/deepspeed-fastgen/README.md]: Thanks!

**Review by loadams — APPROVED** [review]

**Pull request merged**